AI/Nullkiller/Analyzers/BuildAnalyzer.cpp

```diff
@@ -120,6 +120,11 @@ TResources BuildAnalyzer::getTotalResourcesRequired() const
 	return result;
 }
 
+bool BuildAnalyzer::isGoldPreasureHigh() const
+{
+	return goldPreasure > ai->settings->getMaxGoldPreasure();
```

Comment:
Is Preasure meant to be Pressure or Treasure?

Reply:
Pressure